ironfish/src/rpc/adapters/tcpAdapter.test.ts

```diff
@@ -3,64 +3,63 @@
  * file, You can obtain one at https://mozilla.org/MPL/2.0/. */
 /* eslint-disable jest/no-try-expect */
 /* eslint-disable jest/no-conditional-expect */
-import os from 'os'
 import * as yup from 'yup'
-import { IronfishSdk } from '../../sdk'
-import { RpcRequestError, RpcSocketClient } from '../clients'
-import { ALL_API_NAMESPACES, ApiNamespace } from '../routes'
+import { Assert } from '../../assert'
+import { createNodeTest } from '../../testUtilities'
+import { RpcRequestError } from '../clients'
+import { RpcTcpClient } from '../clients/tcpClient'
+import { ALL_API_NAMESPACES, API_NAMESPACES_PROTECTED } from '../routes'
 import { ERROR_CODES, ValidationError } from './errors'
 import { RpcTcpAdapter } from './tcpAdapter'
 
 describe('TcpAdapter', () => {
-  let tcp: RpcTcpAdapter
-  let sdk: IronfishSdk
-  let client: RpcSocketClient
-
-  beforeEach(async () => {
-    const dataDir = os.tmpdir()
-
-    sdk = await IronfishSdk.init({
-      dataDir: dataDir,
-      configOverrides: {
-        enableRpcTcp: true,
-        enableRpcIpc: false,
-        enableRpcTls: false,
-      },
-    })
-
-    const node = await sdk.node()
+  const nodeTest = createNodeTest(false, {
+    config: {
+      enableRpcTcp: true,
+      enableRpcIpc: false,
+      enableRpcTls: false,
+    },
+  })
 
-    tcp = node.rpc.adapters[0] as RpcTcpAdapter
+  let adapter: RpcTcpAdapter
+  let client: RpcTcpClient
 
-    client = sdk.client
+  beforeEach(() => {
+    Assert.isInstanceOf(nodeTest.node.rpc.adapters[0], RpcTcpAdapter)
```

Comment:
very small nit, but I would have flipped this line with the one below and asserted against adapter

Reply:
You can't do that in typescript because you can't assign RpcClient to RpcTcpClient, and then if you change the variable to RpcClient you'll need to copy this assert to every test that uses it.